### ISSUE TITLE
samtools sort truncated file

### ISSUE THREAD

**Issue by paulstretenowich** [GuoliangLi-HZAU/BatMeth2#19]
Hi,

I'm running BatMeth2 doing:
`BatMeth2 align -1 R1.fastq.gz -2 R2.fastq.gz -g reference.fa -of BAM -o pwet -p 30`
and I have the following issue:
`[E::sam_parse1] numeric value out of allowed range`
`samtools sort: truncated file. Aborting`

Thanks in advance for the help,
Paul

**Comment by ZhouQiangwei** [comment]
I'm very sorry, this is a bug in the software. We fixed the bug in the updated software. Please download the latest version and try it?

**Comment by paulstretenowich** [comment]
Hi @ZhouQiangwei the latest release matches with the version I used. Do you want me to try a specific branch of your repo or will you do a new release?

Thanks,
Paul

**Comment by ZhouQiangwei** [comment]
Hi Paul, are you downloading the release version or the latest version of git clone? Is it convenient for you to tell us what species your data are, or to provide data sources for me to test?

**Comment by paulstretenowich** [comment]
Hi @ZhouQiangwei, I used the release and not the clone. As far as I remember I used human genome as reference and it was WGBS reads.
Let me try again one more time and I'll be back to you.

**Comment by ZhouQiangwei** [comment]
Hi Paul, please download the latest version of the software through git clone. The release version has not been updated for a long time. I'm very sorry. If there are still problems with the new version of the software, please feel free to contact us.

**Comment by FAFUshiyan** [comment]
[W::sam_read1] parse error at line 1
[bam_sort_core] truncated file. Aborting.
I used the latest version of the software.
